mapdata/WarcraftLegacies/AbilityData/Skin/1093742657.json

```diff
@@ -0,0 +1,39 @@
+{
+  "OldId": 1667318593,
+  "NewId": 1093742657,
+  "Unk": [
+    0
+  ],
+  "Modifications": [
+    {
+      "Id": 2020631137,
+      "Value": 1
+    },
+    {
+      "Id": 2037408353,
+      "Value": 1
+    },
+    {
+      "Id": 1718840929,
+      "Type": 3,
+      "Value": "(Thrall)"
+    },
+    {
+      "Level": 1,
+      "Id": 828536161,
+      "Type": 3,
+      "Value": "Gives \u003CA01A,DataA1,%\u003E% extra damage to the attacks of nearby friendly units."
+    },
+    {
+      "Id": 1835101793,
+      "Type": 3,
+      "Value": "Warchief\u0027s Command"
```

Comment:
This ability must be named "Command Aura" because it has the same effect as Command Aura.